health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -3030,6 +3024,84 @@
   );
 }
 
+export async function processDataForTargetCalculation(request: any, jsonData: any, codeColumnName: string, localizationMap?: any) {
+  // Retrieve targetConfigs from MDMS
+  const targetConfigs = await searchMDMS([request?.body?.CampaignDetails?.projectType], "HCM-ADMIN-CONSOLE.targetConfigs", request?.body?.RequestInfo);
+
+  // Process each row of the sheet data
+  const resultantData = jsonData.map((row: any) => {
+
+    // Initialize an object to hold row-specific data
+    let rowData: any = { [codeColumnName]: row[codeColumnName] };
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'rowData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)